package/yast2-bootloader.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Jun 29 12:46:58 UTC 2015 - jreidinger@suse.com
+
+- set only proper boot boot flags, otherwise it can confuse some 
```

Comment:
Exactly. I see that the bug report does not help much because you wrote it yourself :) but please try a user's perspective: if the old code failed, the circumstances and symptoms would be such and such. Probably include "Distinguish between MSDOS and GPT tables when setting boot flags" ?